schema/model_pointset_point.json

```diff
@@ -42,6 +42,16 @@
       "description": "Mapping for the point to an internal resource (e.g. BACnet object reference)",
       "type": "string",
       "examples": ["AI3", "400070"]
+    },
+    "translation": {
```

Comment:
erm, not comfortable with this ATM since there's already the concept of translations in the gateway config schema and other places, so I'd want to make sure this is coherent with the overall structure of things.

Reply:
Changed `translation` into `mapping` and an object type, with one property `name`.